pr_review_queue.py

```diff
@@ -171,23 +170,39 @@ def list_green_pull_requests(github_api, org, repo, dry_run):
                 elif "failure" in status or "pending" in status: # requirement 2: CI is a success
                     continue
 
+                assignees = map(lambda x: x["login"], pull_request_details["assignees"] + pull_request_details["requested_reviewers"])
```

Comment:
This should only consider where review is pending, not if they approve or request changes

Reply:
Nvm, that works. If someone already reviewed than they do not show up in `pull_request_details["requested_reviewers"]`